### ISSUE TITLE
[Solarized theme] Code blocks have extra whitespace below them until they're highlighted.

### ISSUE THREAD

**Issue by vr8hub** [typora/typora-issues#1363]
Typora .9.9.13.6 on OSX 10.12.6.

When switching to my theme (a hacked version of Solarized), this is the display of a code block. Notice the apparent extra blank line below the only line in the code block.
![unselected](https://user-images.githubusercontent.com/19378955/38071366-7bacf96c-32e6-11e8-9ed4-03ddac2d38cb.jpg)

If I click in that code block and do nothing but just highlight part of the line, that blank line disappears, as you can see here.
![selected](https://user-images.githubusercontent.com/19378955/38071365-7b95cbde-32e6-11e8-8273-37d01154b912.jpg)

The block should display as it does in the second case (without the apparent blank line) all the time.

This is consistent; it happens every time I switch to another theme and switch back to mine. (The other themes, e.g. Academic, never show that blank line.)

Just for completeness, here's the pertinent section of the CSS:
```css
pre.md-fences {
    padding: 10px 10px;
    margin-bottom: 10px;
}

.md-fences {
    font-family: 'Latin Modern Mono Light', "Latin Modern Mono", monospace !important;
    font-size: 0.9em;
    color: var(--sol-orange);
    background-color: var(--darker-bg-color);
}
```

**Comment by abnerlee** [comment]
I will check the CSS.

It may be a issue of the css theme if you do not meet such bug in other themes

**Comment by vr8hub** [comment]
It certainly might be, but I've commented and uncommented out things in an effort to confirm it isn't. And I don't know how the CSS could make something look one way, then look another when it has focus, then *keep* that way when it doesn't have focus. If it was always with the blank line when not in focus and always without when in focus, that would be one thing. But once the blank line disappears, it stays gone. Until the file is opened again, or the theme is changed and changed back.

Here's the full CSS I'm using. The fonts are the fonts from Academic, I just copied them to the solarized directory.
[solarized.zip](https://github.com/typora/typora-issues/files/1860799/solarized.zip)
